common/term.go

```diff
@@ -0,0 +1,11 @@
+package common
+
+import (
+	"os"
+	"strings"
+)
+
+func Is256ColorSupported() bool {
+	term := os.Getenv("TERM")
+	return strings.Contains(term, "256color")
+}
```

Comment:
I'm sorry but this check is way too naive

Take a look at project that could help you to detect that capabilities

https://pkg.go.dev/github.com/gookit/color#IsSupport256Color

Reply:
The simplest would be to use this 

https://pkg.go.dev/github.com/charmbracelet/lipgloss#DefaultRenderer

The call ColorProfile

Reply:
Thank you for pointing out the issues! I modify this part of the code later.

Reply:
I opened
 - #https://github.com/hengyoush/kyanos/issues/186